cms/envs/aws.py

```diff
@@ -185,6 +185,7 @@
 #theming start:
 PLATFORM_NAME = ENV_TOKENS.get('PLATFORM_NAME', 'edX')
 STUDIO_NAME = ENV_TOKENS.get('STUDIO_NAME', 'edX Studio')
+TENDER_DOMAIN = ENV_TOKENS.get('TENDER_DOMAIN', None)
```

Comment:
The should fall back to `TENDER_DOMAIN` defined in common.py see `LANGUAGE_CODE` for an example.